pipelines/test/kwok/cluster.yaml

```diff
@@ -45,3 +51,11 @@ pipeline:
 
       kubectl wait --for=condition=Ready nodes --all
       kubectl cluster-info
+      export KUBERNETES_SERVICE_HOST="127.0.0.1"
+      export KUBERNETES_SERVICE_PORT="32764"
+      mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
+      CA=$(kubectl config view --raw -o jsonpath='{.clusters[0].cluster.certificate-authority}')
+      cp $CA /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+      echo ${{inputs.namespace}} > /var/run/secrets/kubernetes.io/serviceaccount/namespace
+      kubectl create sa ${{inputs.namespace}}
+      kubectl create token ${{inputs.namespace}} > /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
I'd not create the token by default following k8s. For security reason, token are not created by default since 1.23, and they're not always required in our tests IMHO